src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import duke.exceptions.EmptyCommandException;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static void response(String line, TaskManager action) {
+        String[] words = line.split(" ");
+        String commandWord = words[0];
+        int length = commandWord.length();
+        String taskLine = " ";
+        if (line.length() > length) {
```

Comment:
Consider extracting the logic into another method (i.e. SLAP more) to make code easier to read and understand.